chia/server/address_manager.py

```diff
@@ -159,26 +160,32 @@ def get_selection_chance(self, now: Optional[int] = None) -> float:
         return chance
 
 
+def create_tried_matrix() -> list[list[int]]:
+    return [[-1 for x in range(BUCKET_SIZE)] for y in range(TRIED_BUCKET_COUNT)]
+
+
+def create_new_matrix() -> list[list[int]]:
+    return [[-1 for x in range(BUCKET_SIZE)] for y in range(NEW_BUCKET_COUNT)]
+
+
 # This is a Python port from 'CAddrMan' class from Bitcoin core code.
+@dataclass
 class AddressManager:
-    id_count: int
-    key: int
-    random_pos: list[int]
-    tried_matrix: list[list[int]]
-    new_matrix: list[list[int]]
-    tried_count: int
-    new_count: int
-    map_addr: dict[str, int]
-    map_info: dict[int, ExtendedPeerInfo]
-    last_good: int
-    tried_collisions: list[int]
-    used_new_matrix_positions: set[tuple[int, int]]
-    used_tried_matrix_positions: set[tuple[int, int]]
-    allow_private_subnets: bool
-
-    def __init__(self) -> None:
-        self.clear()
-        self.lock: Lock = Lock()
+    id_count: int = 0
+    key: int = randbits(256)
```

Comment:
```suggestion
    key: int = field(default_factory=lambda: randbits(256))
```
or
```suggestion
    key: int = field(default_factory=functools.partial(randbits, 256))
```
depending your preference